--- linux-4.10-rc7/drivers/gpu/drm/panel/panel-simple.c	2017-02-06 00:10:58.000000000 +0100
+++ linux-4.10-rc7.mod/drivers/gpu/drm/panel/panel-simple.c	2017-02-13 11:58:15.419000707 +0100
@@ -299,6 +299,8 @@
 	struct panel_simple *panel;
 	int err;
 
+	dev_info(dev, "panel_simple_probe for %s", dev->of_node? dev->of_node->full_name : "no-of-node");
+
 	panel = devm_kzalloc(dev, sizeof(*panel), GFP_KERNEL);
 	if (!panel)
 		return -ENOMEM;
@@ -321,19 +323,24 @@
 
 	backlight = of_parse_phandle(dev->of_node, "backlight", 0);
 	if (backlight) {
+		dev_info(dev, "Trying backlight: %s", backlight->full_name);
 		panel->backlight = of_find_backlight_by_node(backlight);
 		of_node_put(backlight);
 
-		if (!panel->backlight)
+		if (!panel->backlight) {
+			dev_info(dev, "No backlight yet");
 			return -EPROBE_DEFER;
+		}
 	}
 
 	ddc = of_parse_phandle(dev->of_node, "ddc-i2c-bus", 0);
 	if (ddc) {
+		dev_info(dev, "Trying ddc %s", ddc->full_name);
 		panel->ddc = of_find_i2c_adapter_by_node(ddc);
 		of_node_put(ddc);
 
 		if (!panel->ddc) {
+			dev_info(dev, "ddc not found");
 			err = -EPROBE_DEFER;
 			goto free_backlight;
 		}
@@ -343,9 +350,12 @@
 	panel->base.dev = dev;
 	panel->base.funcs = &panel_simple_funcs;
 
+	dev_info(dev, "adding panel");
 	err = drm_panel_add(&panel->base);
-	if (err < 0)
+	if (err < 0) {
+		dev_info(dev, "drm_panel_add failed");
 		goto free_ddc;
+	}
 
 	dev_set_drvdata(dev, panel);
 
@@ -1722,9 +1759,12 @@
 {
 	const struct of_device_id *id;
 
+	dev_info(&pdev->dev, "%s for %s", __func__, pdev->dev.of_node->full_name);
 	id = of_match_node(platform_of_match, pdev->dev.of_node);
-	if (!id)
+	if (!id) {
+		dev_info(&pdev->dev, "%s no of_match_node match", __func__);
 		return -ENODEV;
+	}
 
 	return panel_simple_probe(&pdev->dev, id->data);
 }
