--- linux-4.10-rc7/drivers/gpu/drm/panel/panel-simple.c	2017-02-06 00:10:58.000000000 +0100
+++ linux-4.10-rc7.mod/drivers/gpu/drm/panel/panel-simple.c	2017-02-13 11:58:15.419000707 +0100
@@ -2079,6 +2079,34 @@
 	.lanes = 4,
 };
 
+static const struct drm_display_mode nlt_192108AC18_mode = {
+	.clock = 74175 * 2,
+	.hdisplay = 960 * 2,
+	.hsync_start = (960 + 48) * 2,
+	.hsync_end = (960 + 48 + 44) * 2,
+	.htotal = (960 + 48 + 44 + 48) * 2,
+	.vdisplay = 1080,
+	.vsync_start = 1080 + 15,
+	.vsync_end = 1080 + 15 + 15,
+	.vtotal = 1080 + 15 + 15 + 15,
+	.vrefresh = 60,
+};
+
+static const struct panel_desc_dsi nlt_192108AC18 = {
+	.desc = {
+		.modes = &nlt_192108AC18_mode,
+		.num_modes = 1,
+		.bpc = 8,
+		.size = {
+			.width = 344,
+			.height = 193,
+		},
+	},
+	.flags = MIPI_DSI_MODE_VIDEO,
+	.format = MIPI_DSI_FMT_RGB888,
+	.lanes = 4,
+};
+
 static const struct of_device_id dsi_of_match[] = {
 	{
 		.compatible = "auo,b080uan01",
@@ -2096,6 +2124,9 @@
 		.compatible = "panasonic,vvx10f004b00",
 		.data = &panasonic_vvx10f004b00
 	}, {
+		.compatible = "nlt,192108AC18",
+		.data = &nlt_192108AC18
+	}, {
 		/* sentinel */
 	}
 };
